db/seeds.rb

```diff
@@ -557,6 +557,23 @@
 
 ##############################################################################
 
+seeder.create_if_none(DisplayAd) do
+  orgs_in_random_order = Organization.order(Arel.sql("RANDOM()"))
+  orgs = orgs_in_random_order.to_a
+
+  DisplayAd::ALLOWED_PLACEMENT_AREAS.each do |placement_area|
+    DisplayAd.create!(
+      name: "#{Faker::Lorem.word} #{placement_area}",
+      body_markdown: Faker::Lorem.sentence,
+      published: true,
+      approved: true,
+      placement_area: placement_area
+    )
+  end
```

Comment:
@Ridhwana The difference now is I choose to loop through the all ALLOWED_PLACEMENT_AREAS.